examples/seismic/model.py

```diff
@@ -503,3 +561,64 @@ def vp(self, vp):
             initialize_function(self.m, 1 / (self.vp * self.vp), self.nbpml)
         else:
             self.m.data = 1 / vp**2
+
+
+class ModelElastic(Pysical_Model):
```

Comment:
should we actually also split `Model` into `ModelIsotropic` and `ModelAnistropic`? if so, it clearly doesn't need to be in this PR

Reply:
>should we actually also split Model into ModelIsotropic and ModelAnistropic?

No these two have the same parametrization with some to 0 while the elastic one has a different parametrization.

Reply:
👍